src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -7081,6 +7081,68 @@ public class Methods2
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public void SharedProjectFieldAndPropertiesTreatedAsIdentical()
```

Comment:
Do we not have infrastructure to test this in VB? The bug has a ready-made test case. Seems like a simple thing to create a unit test around so it's provably testing against regressions. I'd like to have that test in place so that when I refactor C# and VB symbol completion, I'm sure that I don't regress this behavior in either language.